Lars-02/Smoelenboek
# Feature/profile side nav

# Create Profile Navigation Component

## Description
This PR is made for the profile side navigation component. It includes 4 menu items, which will be used to navigate between tabs on the profile. One menu item has a @click option to handle the show/hide of the extra information tab.

## Testing
1. Open your project
2. Run composer via terminal
3. Run command in terminal `php artisan migrate:fresh --seed` to clear the database.
4. Run command `php artisan serve`
5. Place the code snippet in this PR inside resources/views/employee/form.blade.php. The current code in the file can be replaced for this test. 
`<x-layout>`
    `<div x-data="{show: false}">`
        `<div>`
            `<x-profilenavigation href-account="#" href-work-hours="#" href-delete="#" click-action="{ show = !show }">`
`</x-profilenavigation>`
            `<x-card title="Account">`
            `</x-card>`
           `<div>`
               `<div x-show="show">`
                    `<x-card title="Minoren, Afdeling en Expertises">`
                    `</x-card>`
                `</div>`
            `</div>`
        `</div>`
    `</div>`
`</x-layout>`
6. Log in as the test@avans.nl user. You will automatically redirected to the user registration page.
7. Toggle this button to see the effect:
![image](https://user-images.githubusercontent.com/55989826/112011915-02b0e700-8b29-11eb-960a-946e8334a299.png)

After testing discard your changes in the resources/views/employee/form.blade.php.
NOW YOU'RE DONE!
